docs/use-cases/wallet.md

```diff
@@ -0,0 +1,140 @@
+
+### 1.2 Requirements
+
+#### User Personas 
+
+1. Customers - participants and active users of a given cryptocurrency wallet.
+2. Integrator - development and management staff of a wallet that evaluates the requirements and delivers the integration with the OMG Network.
+3. OMG Network - a technical and management team within the OMG Network that supports the Integrator during the integration process.
+
+#### User Stories
+
+1. As a customer, I can deposit ERC20 tokens from the Ethereum network onto the OMG Network without leaving my cryptocurrency wallet.
+2. As a customer, I can send ERC20 tokens that have been deposited to my wallet to any OMG Network compatible wallet.
+3. As a customer, I can withdraw deposited ERC20 tokens from my wallet back to the Ethereum network.
+
+## 2. Proposed design
+
+The proposed design is based on the lessons learned during the development of the OMG Network mobile wallet, as well as extensive research, interviews, and customer development. The information provided here should be treated as a general suggestion, instead of clear rules that you absolutely must follow.
+
+### 2.1 Integration scope
+
+The scope for integrating with the OMG Network by a given cryptocurrency wallet includes several steps. The current scope is very similar to the [Exchange Use Case Integration Scope](https://docs.omg.network/use-cases/exchange#21-integration-scope), thus some parts will be used as references to that use case to avoid redundant information.
+
+There are two ways a wallet can integrate with the OMG Network:
+1. Natively: separate implementations for Android and iOS using Java/Kotlin or Objective-C/Swift respectively.
+2. Hybrid: single implementation based on React Native.
+
+At the moment, we do not provide SDKs for Android and iOS yet, thus the first option can be more complicated because some of the Watcher Info API endpoints may not be available, and we cannot guarantee that external dependencies (web3, crypto, encoding/decoding utils, etc.) will behave the same way as they do with JavaScript libraries.
+
+We do provide React Native support for [omg-js](https://github.com/omgnetwork/omg-js) so you shouldn't have any issues using this approach. If possible, we suggest using the latter option because it is well tested by our team.
+
+#### 2.1.1 Query blockchain
+
+Querying blockchain is the most basic thing every integrator needs to know how to do. A wallet provider might be interested in retrieving data about transactions, accounts, deposits, and exits. You can find the full list of commands for this step in the [Exchange Use Case](https://docs.omg.network/use-cases/exchange#212-query-blockchain).
+
+*Implementation details:*
+
+The information about the token and its price currently have to be retrieved from different sources (Ethereum node, such as Infura, and one of the public price feeds, such as ...), which makes it more difficult to display the full token details and the balance in different fiat currencies.
```

Comment:
hmm we don't have the price feeds here, - I would suggest coinmarketcap, coin-gecko or Amber data?

Reply:
on that note, each native wallet would already have the price feed integrated to the core application I think, we could just tell them to use whatever they currently use